src/constants/addresses.ts

```diff
@@ -22,6 +22,15 @@ const CELO_QUOTER_ADDRESSES = '0x82825d0554fA07f7FC52Ab63c961F330fdEFa8E8'
 const CELO_NONFUNGIBLE_POSITION_MANAGER_ADDRESSES = '0x3d79EdAaBC0EaB6F08ED885C05Fc0B014290D95A'
 const CELO_TICK_LENS_ADDRESSES = '0x5f115D9113F88e0a0Db1b5033D90D4a9690AcD3D'
 
+// bsc v3 addresses
```

Comment:
Confusing that we use BNB and BSC interchangeably.

Reply:
the pr was initially made w/ both, and I've tried to transition to just using BNB. Thanks for catching this.